src/i18n/rpi-imager_it.ts

```diff
@@ -216,64 +216,64 @@ Aggiungi sia &apos;rpi-imager.exe&apos; che &apos;fat32format.exe&apos; all&apos
     <message>
         <location filename="../DstPopup.qml" line="24"/>
         <source>Storage</source>
-        <translation type="unfinished">Scheda SD</translation>
+        <translation>Storage</translation>
     </message>
     <message>
         <location filename="../DstPopup.qml" line="38"/>
         <source>No storage devices found</source>
-        <translation type="unfinished">Nessun dispositivo archiviazione trovato</translation>
+        <translation>Nessun dispositivo archiviazione trovato</translation>
     </message>
     <message>
         <location filename="../DstPopup.qml" line="67"/>
         <source>Exclude System Drives</source>
-        <translation type="unfinished"></translation>
+        <translation>Escludi unità sistema</translation>
     </message>
     <message>
         <location filename="../DstPopup.qml" line="94"/>
         <source>gigabytes</source>
-        <translation type="unfinished"></translation>
+        <translation>Gigabyte</translation>
     </message>
     <message>
         <location filename="../DstPopup.qml" line="96"/>
         <location filename="../DstPopup.qml" line="152"/>
         <source>Mounted as %1</source>
-        <translation type="unfinished">Montato come %1</translation>
+        <translation>Montato come %1</translation>
     </message>
     <message>
         <location filename="../DstPopup.qml" line="139"/>
         <source>GB</source>
-        <translation type="unfinished"></translation>
+        <translation></translation>
```

Comment:
Still an empty translation-string here? :confused:

Reply:
Sorry but in my fiule is

Quite strange. 
QTlinguist report as translated.
I will fix this.,

Reply:
The ` type="unfinished"` doesn't matter - it's in red, which means it's the old version of the line. It's only the `<translation></translation>` in the green (new) version of the line that I was worried about.